CybeReason/malop/ingest/parser.yml

```
@@ -0,0 +1,105 @@
+name: cybereason-malop
+pipeline:
+  - name: parsed_event
+    external:
+      name: cef.parse-cef
+  - name: parsed_timestamp
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{parsed_event.message.rt | float / 1000}}'
+        output_field: datetime
+        format: timestamp
+    filter: '{{parsed_event.message.rt != null}}'
+  - name: parsed_modified_at
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{parsed_event.message.ModifiedTime | float / 1000}}'
+        output_field: datetime
+        format: timestamp
+    filter: '{{parsed_event.message.ModifiedTime != null}}'
+  - name: set_ecs_fields
+  - name: set_cybereason_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          '@timestamp': '{{parsed_timestamp.datetime}}'
+        filter: '{{parsed_event.message.rt != null}}' 
+      - set:
+          observer.vendor: '{{parsed_event.message.DeviceVendor}}'
+          observer.product: '{{parsed_event.message.DeviceProduct}}'
+          observer.version: '{{parsed_event.message.DeviceVersion}}'
+      - set:
+          event.action: '{{parsed_event.message.Name}}'
+          event.severity: '{{parsed_event.message.Severity}}'
+          event.code: '{{parsed_event.message.DeviceEventClassID}}'
+          event.type: 'info'
+          event.id: '{{parsed_event.message.CybereasonCEFgeneratorBatchId}}'
```

We cannot set this field to `event.id`. It is used in the workflow and we lost event traceability by erasing it's value.
```suggestion
          cybereason.event.id: '{{parsed_event.message.CybereasonCEFgeneratorBatchId}}'
```

---

fixed